schemas/iso19139/src/main/plugin/iso19139/loc/eng/labels.xml

```diff
@@ -3552,4 +3552,8 @@
     <label>Upper cardinality</label>
     <description>Upper cardinality</description>
   </element>
+  <element name="xlink:type">
+    <label>Link</label>
```

Comment:
Maybe the label should be `type` or `link type` and the element added to all languages ?

Reply:
Done, thanks for the feedback